src/Framework/src/Microsoft.AspNetCore.App.Runtime.csproj

```diff
@@ -448,4 +450,19 @@ This package is an internal implementation of the .NET Core SDK and is not meant
       Condition="'$(ArchiveExtension)' == '.tar.gz'" />
   </Target>
 
+  <Target Name="IncludeFrameworkListFile"
```

Comment:
Is there a way to shared the implementation of this target between the runtime pack and ref pack? Just to reduce duplication, not strictly necessary.

Reply:
I think it would cause more pain in the long run than help. Differences include RefPackContent vs SharedFxContent, the target file name, different DependsOnTargets, etc. Anyways, all this target is doing is invoking another target and putting stuff into package files.